scripts/ci_build.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+set -xeuo pipefail
+
+script_dir=$(dirname ${BASH_SOURCE[0]})
+source ${script_dir}/ci_build_common.sh
+
+# Kill any lingering Gradle workers
+ps auxwww | grep Gradle | grep -v grep | awk '{ print $2; }' | xargs -r kill || true
```

Comment:
Remove this as it's very old from the previous team, and I'm not sure if it's valid.